be/src/storage/rowset/rowset.cpp

```diff
@@ -446,6 +486,43 @@ Status Rowset::copy_files_to(KVStore* kvstore, const std::string& dir) {
             return Status::IOError(fmt::format("Error to copy file. src: {}, dst: {}, error:{} ", src_path, dst_path,
                                                std::strerror(Errno::no())));
         }
+        // copy index
+        const auto& indexes = *_schema->indexes();
+        if (!indexes.empty()) {
+            for (const auto& index : indexes) {
+                if (index.index_type() == IndexType::GIN) {
+                    std::string dst_index_path = IndexDescriptor::inverted_index_file_path(dir, rowset_id().to_string(),
+                                                                                           i, index.index_id());
+                    if (fs::path_exist(dst_index_path)) {
+                        LOG(WARNING) << "Index path already exist: " << dst_path;
+                        return Status::AlreadyExist(fmt::format("Index path already exist: {}", dst_path));
+                    }
+
+                    std::string src_index_path = IndexDescriptor::inverted_index_file_path(
+                            _rowset_path, rowset_id().to_string(), i, index.index_id());
+
+                    if (!fs::create_directories(dst_index_path).ok()) {
+                        LOG(WARNING) << "Error to create inverted index dir " << dst_index_path
+                                     << ". errno=" << std::strerror(Errno::no());
+                        return Status::IOError(fmt::format("Error to create inverted index dir {}", dst_index_path));
+                    }
+                    std::set<std::string> files;
+                    RETURN_IF_ERROR(fs::list_dirs_files(src_index_path, nullptr, &files));
+                    for (const auto& file : files) {
+                        auto src_absolute_path = fmt::format("{}/{}", src_index_path, file);
+                        auto dst_absolute_path = fmt::format("{}/{}", dst_index_path, file);
+
+                        if (!fs::copy_file(src_index_path, dst_index_path).ok()) {
```

Comment:
It seems that it should copy_file using src_absolute_path and dst_absolute_path?